doc/whats_new.rst

```diff
@@ -30,6 +30,7 @@ Bug
 
 API
 ~~~
+- Enforce that the name arg is a required field for :func`mne_bids.make_dataset_description`, by `Teon Brooks`_(`#279 <https://github.com/mne-tools/mne-bids/pull/279>`)
```

Comment:
I think this should go in the Bug section since this is not a backwards compatible change